### DISCUSSION TITLE
Collect raw images and Roi from users community

### DISCUSSION THREAD

**Discussion by nliaudat** [jomjol/AI-on-the-edge-device#1998] [Ideas]
There is no meter collection with labels available on the web.

With the great users community of AI-on-the-edge-device, it could be easy to collect such information and perhaps train a new AI model which extract counter region and digits area. 

If it works, it could replace the difficult step of ROI configuration. If not, it could attract universities and students to this project or be usefull for others. 

@haverland @jomjol @caco3 : What do you think of that proposal ?

The idea come from 
[4694-2020.pdf](https://github.com/jomjol/AI-on-the-edge-device/files/10689050/4694-2020.pdf)

**Comment by caco3** [comment]
Is the configuration of the ROIs so difficult?
Once you understand how to do it it is fairly easy. And it anyway only has to be done once.

I am not against your idea, but for me I would not give it high priority.

**Comment by jomjol** [comment]
I think, this would be a bit too much for the ESP32 calculation capabilities and the effort to do so is very high, including training data.

**Comment by ghost** [comment]
> There is no meter collection with labels available on the web.
> 
> With the great users community of AI-on-the-edge-device, it could be easy to collect such information and perhaps train a new AI model which extract counter region and digits area.
> 
> If it works, it could replace the difficult step of ROI configuration. If not, it could attract universities and students to this project or be usefull for others.
> 
> @haverland @jomjol @caco3 : What do you think of that proposal ?
> 
> The idea come from [4694-2020.pdf](https://github.com/jomjol/AI-on-the-edge-device/files/10689050/4694-2020.pdf)

Yolos are much bigger and would have the need to recognize the whole image instead of small parts. A complete 600x800 image is not possible to compute on esp32 especially the RAM is to small.

Here ones with Yolo V4 https://betterprogramming.pub/yolov4-for-water-meter-reading-5d27191bc053 

I imagined about an automatic setup with yolo in a browser a time before, but it would need much of full meter images. And the meter images has serial numbers on it. So it is not easy to get anonymised data.
